example/settings.py

```diff
@@ -102,7 +106,7 @@
 
 ES_CONNECTIONS = {
     "default": {
-        "hosts": [os.environ.get('ELASTICSEARCH_HOST', 'localhost')],
+        "hosts": [os.environ.get('ELASTICSEARCH_HOST', 'elasticsearch')],
```

Comment:
This will break existing user setups if they are expecting to run "elasticsearch" on localhost, so you should note this in the README.

I've been hesitant to make this change because the wonderful Docker for Mac will make this unnecessary, as you can address docker containers via `localhost`.